shared/elements/gaia-header/dist/gaia-header.js

```diff
@@ -1050,6 +1055,42 @@ module.exports = component.register('gaia-header', {
     setTimeout(() => this.dispatchEvent(e));
   },
 
+  // save changes form on editing a contact
```

Comment:
This is wrong solution, because it adds application-specific functionality into the component which is common for entire platform. All application-specific logic shall be implemented in scope of application e.g. in handler of ``'action'`` event dispatched from ``onActionButtonClick`` handler or somehow else.